python-package/xgboost/core.py

```diff
@@ -306,11 +306,12 @@ def _check_distributed_params(kwargs: Dict[str, Any]) -> None:
         raise TypeError(msg)
 
     if device and device.find(":") != -1:
-        raise ValueError(
-            "Distributed training doesn't support selecting device ordinal as GPUs are"
-            " managed by the distributed frameworks. use `device=cuda` or `device=gpu`"
-            " instead."
-        )
+        if device != "sycl:gpu":
```

Comment:
What happens if it's sycl GPU, is dask going to select the appropriate GPU?

Reply:
Yes, the appropriate GPUs would be selected

Reply:
Got it.